RELEASE/scripts/autoscend/autoscend_header.ash

```diff
@@ -1089,6 +1082,35 @@ int freeRestsRemaining();
 boolean uneffect(effect toRemove);
 boolean useCocoon();
```

Comment:
Didn't we get rid of this function?

Reply:
we got rid of in restore branch which has not been merged yet. i am going to have to manually resolve this conflict later